There is no corresponding catch statement for the throw.
